### ISSUE TITLE
High resolution training

### ISSUE THREAD

**Issue by trqminh** [Gal4way/TPD#18]
Hi @Gal4way, 
Thank you for publishing your work.

I am using this repository and trying to train on higher resolution than the default config. In specific, I changed the image size to 1024x768. I am using the same config setup, I only change the batch size per gpu to 2 to avoid the out-of-memory issue.
The training results look quite bad (I added some examples).
![00006_00_grid](https://github.com/user-attachments/assets/e56c97f9-79d6-40f0-bf63-91f4f62aaec6)
![00008_00_grid](https://github.com/user-attachments/assets/a35eca98-c104-4f20-8ca1-3d6f0d05b9b2)

I am not sure what could be the problem so I would like to ask if you have any idea what the problem could be?

Thank you for your consideration.

**Comment by Spritan** [comment]
@trqminh i am also trying to do the same, if you dont mind can you tell me, what is is the vram consumption at batch size = 2. for 1024x768

**Comment by Gal4way** [comment]
I noticed that the model's predicted masks are inaccurate and the generated try-on images don't look good. I think this might be because the training steps weren't enough, so the model hasn't fully converged yet. The reduced batch size could also be a factor. But I recommend training for more epochs first to rule out the possibility of insufficient training.

**Comment by qbl168** [comment]
Hi @trqminh, did you manage to get good results while training with 1024x768 images?
